DesignSafe-CI/portal
# fix: DES-2765 user may interpret missing app variant label is fixed by CMS

## Overview: ##

Change help text for app variant label to avoid a misinterpretation that every app listing will use app entry/bundle label as fallback if the app variant has no label defined.

## PR Status: ##

* Work in Progress.
* Hold.

## Related: ##

* [DES-2765](https://tacc-main.atlassian.net/browse/DES-2765)
* alternative to https://github.com/DesignSafe-CI/portal/pull/1230

## Summary of Changes: ##

- **changed** help text of App variant "Label"
- **added** model migration

> [!NOTE]
> **Alternative:** [Customize & Translate Text](https://github.com/TACC/Core-CMS/blob/v4.11.0/docs/develop-project.md#customize--translate-text), but I haven't tried this on DesignSafe. _I don't want to fight unexpected obstacles unless this text change is a good idea._

## Testing Steps: ##
1. Run Portal.
2. `docker exec -it des_django bash`
3. `./manage.py migrate`
4. Edit an App entry.
5. Verify text is changed.

> [!IMPORTANT]
> I did not test this. I fear of putting my DesignSafe in an irreversible state.

## UI Photos: ##

| before | expected (untested)<br /><sup>(hacked screenshot)</sup> |
| - | - |
| ![before](https://github.com/DesignSafe-CI/portal/assets/62723358/e1d2de0a-9461-4551-bcc1-3794ceeb9c4a) | ![expectation](https://github.com/DesignSafe-CI/portal/assets/62723358/3e0e397c-44c5-4261-910f-9cc607558e49) |
